k8s.io/configmap-nginx.yaml

```diff
@@ -244,6 +244,8 @@ data:
           rewrite ^/test-health$     http://velodrome.k8s.io/dashboard/db/bigquery-metrics redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-gubernator/triage/index.html redirect;
+          rewrite ^/youtube$          https://www.youtube.com/c/kubernetescommunity redirect;
```

Comment:
Sorry, I wasn't clear - we don't need a DNS name *and* a go link.  One or the other :)